packages/experience/src/hooks/use-session-storages.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Provides a hook to access the session storage.
+ */
+import { type SsoConnectorMetadata } from '@logto/schemas';
+import { trySafe } from '@silverhand/essentials';
+import { useCallback } from 'react';
+
+const logtoStorageKeyPrefix = `logto:${window.location.origin}`;
```

Comment:
Should we include tenant ID in the key prefix?

Reply:
no need , origin contains tenant id